leshan-client-core/src/main/java/org/eclipse/leshan/client/object/Security.java

```diff
@@ -108,6 +109,14 @@ public static Security noSec(String serverUri, int shortServerId) {
         return new Security(serverUri, false, SecurityMode.NO_SEC.code, new byte[0], new byte[0], new byte[0],
                 shortServerId);
     }
+    
+    /**
+     * Returns a new security instance (OSCORE) for a device management server.
+     */
+    public static Security secOSCore(String serverUri, int shortServerId) {
```

Comment:
here this would be more something like :
- oscore (or oscoreOnly)

and one day we will add something like this ? 
- pskWithOscore 
- rpkWithOscore
- x509WithOscore

Reply:
Yes, that makes sense. I can rename the current method to oscore/oscoreOnly. And then we proceed with adding methods later that combine DTLS+OSCORE.

Reply:
Renamed the current method to oscoreOnly in commit dd58e43e1da396aa35c54a24b898752647c04bdd . In the future further methods can be added.